examples/kubernetes/static_provisioning/README.md

```diff
@@ -4,7 +4,6 @@ This example shows how to make a static provisioned Mountpoint for S3 persistent
 ## Examples in this folder
 - `static_provisioning.yaml` - spawning a pod which creates a file with name as the current date/time
 - `non_root.yaml` - same as above, but the pod is spawned as non-root (uid `1000`, gid `2000`)
-- `s3_express_specify_az.yaml` - same as above, but this uses a [S3 Express One Zone](https://docs.aws.amazon.com/AmazonS3/latest/userguide/s3-express-one-zone.html) directory bucket and shows how to specify the availability zone (AZ) of the [persistent volume](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity) to co-locate the pods with the bucket for lower latency access
```

Comment:
I deleted this but we will have a new readme, just want to make sure my agent doesn't consider this.